hw/ip/rstmgr/data/rstmgr_sec_cm_testplan.hjson

```diff
@@ -27,43 +27,55 @@
       name: sec_cm_bus_integrity
       desc: "Verify the countermeasure(s) BUS.INTEGRITY."
       milestone: V2S
-      tests: []
+      tests: ["rstmgr_tl_intg_err"]
     }
     {
       name: sec_cm_scan_intersig_mubi
       desc: "Verify the countermeasure(s) SCAN.INTERSIG.MUBI."
       milestone: V2S
-      tests: []
+      tests: ["rstmgr_sec_cm_scan_intersig_mubi"]
     }
     {
       name: sec_cm_leaf_rst_bkgn_chk
-      desc: "Verify the countermeasure(s) LEAF.RST.BKGN_CHK."
+      desc: "Verify the countermeasure(s) LEAF.RST.BKGN_CHK. XXX need more info"
       milestone: V2S
       tests: []
     }
     {
       name: sec_cm_leaf_rst_shadow
-      desc: "Verify the countermeasure(s) LEAF.RST.SHADOW."
+      desc: "Verify the countermeasure(s) LEAF.RST.SHADOW. XXX need more info"
       milestone: V2S
       tests: []
     }
     {
       name: sec_cm_leaf_fsm_sparse
-      desc: "Verify the countermeasure(s) LEAF.FSM.SPARSE."
+      desc: '''Verify the countermeasure(s) LEAF.FSM.SPARSE.
+
+            Force leaf rst check state to illegal value.
+            This is triggered by common cm primitives
+            '''
       milestone: V2S
-      tests: []
+      tests: ["rstmgr_sec_cm"]
```

Comment:
@weicaiyang 
I don't think this test verifies this countermeasure.

Reply:
> Sparsely encoded fsm for each leaf rst check. The Hamming delta is only 3 as there are a significant number of leaf resets

From the name and description, this looks like a sparse FSM. If so, it should be able to verify it.